src/irr/pch.h

```diff
@@ -1,3 +1,7 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
```

Comment:
@Przemog1 given the new PCH Cmake stuff, do we even need these headers anymore?

Reply:
I don't remember actually..

Reply:
check.

Reply:
TODO

Reply:
it has to stay